ui/src/components/Layout/Sidebar.vue

```diff
@@ -26,5 +26,9 @@ const isAdmin = computed(() => store.getters['authModule/isAdmin'])
   background: var(--feather-secondary);
   z-index: 1;
 }
+
+.feather-styles a:hover{
```

Comment:
I understand the feather-styles class on the root is in the docs, but there has to be a better way to do this particular change. This will remove any text link from having an underline across the entire site. I'm also not a fan of !important if it can be avoided at any cost.

If you really do need these css, move it into a global file somewhere near the root of the project, and find a way to make it work without the important (if you add more specificity to the class paths, you shouldn't need the important).